packages/headless/src/controllers/facets/range-facet/date-facet/headless-date-facet.ts

```diff
@@ -86,6 +89,13 @@ export interface DateFacet extends Controller {
    */
   toggleSelect(selection: DateFacetValue): void;
 
+  /**
+   * Toggles the specified facet value, deselecting others.
```

Comment:
Will need to confirm with style and language committee regarding deselect vs unselect (or clear according to microsoft...). If there needs to be a term replace in multiple locations, I'll make a separate PR.